modules/virt-migrating-bulk-vms-different-storage-class-web.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * virt/managing_vms/virt-migrating-vms-in-single-cluster-to-different-storage-class.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-migrating-bulk-vms-different-storage-class-web_{context}"]
+= Migrating VMs in a single cluster to a different storage class by using the web console
+
+By using the {product-title} web console, you can migrate single-cluster VMs in bulk from one storage class to another storage class.
+
+.Prerequisites
+
+* The VMs you select for each bulk migration must be in the same namespace.
+* The Migration Toolkit for Containers (MTC) must be installed.
+
+.Procedure
+
+. From the {product-title} web console, navigate to *Virtualization* → *VirtualMachines*.
+
+. From the list of VMs, select each VM you want to move from its current storage class.
+
+. Select *Actions* -> *Migrate storage*.
++
+You can also access this option by opening the Options menu {kebab} for a selected VM, then selecting *Migration* -> *Storage*.
++
+The *Migrate VirtualMachine storage* page opens.
+
+. To review the VMs you want to migrate, click the link that identifies the number of VMs and volumes. Click *View more* to see the full list.
+
+. Select either the entire VM or only selected volumes for storage class migration. If you choose to migrate only selected volumes, the page expands to allow you to make specific selections.
++
+You can also click *VirtualMachine name* to select all VMs.
+
+. Click *Next*.
+
+. From the list of available storage classes, select the destination storage for the migration.
```

Comment:
_destination storage_ sounds a bit weird to me. 

```suggestion
. From the list of available storage classes, select the destination storage classes for the migration.
```
```suggestion
. From the list of available storage classes, select the destination for the migration.
```

Reply:
`destination storage class`

Reply:
Used Jenia's suggestion